src/content/header/header.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: The OG tokenization protocol on Bitcoin
+title: The Original Bitcoin Layer-2 Protocol
```

Comment:
Maybe "Original DeFi and NFT Protocol on Bitcoin"?

Reply:
Layer 2 is in contradiction with Protocol